webapp/components/interop.js

```diff
@@ -429,182 +502,228 @@ class InteropDashboard extends PolymerElement {
+                    <template is="dom-repeat" items="{{section.rows}}" as="rowName">
+                      <tr data-feature$="[[rowName]]">
+                        <td>
+                          <a href$="[[getRowInfo(rowName, 'tests')]]">[[getRowInfo(rowName, 'description')]]</a>
+                        </td>
+                        <td>[[getBrowserScoreForFeature(0, rowName, stable)]]</td>
+                        <td>[[getBrowserScoreForFeature(1, rowName, stable)]]</td>
+                        <td>[[getBrowserScoreForFeature(2, rowName, stable)]]</td>
+                        <td>[[getBrowserScoreForFeature(3, rowName, stable)]]</td>
+                      </tr>
+                    </template>
                     <tr class="subtotal-row">
-                      <td><strong>SUBTOTAL</strong></td>
+                      <td><strong>TOTAL</strong></td>
                       <td>[[getSubtotalScore(0, section, stable)]]</td>
                       <td>[[getSubtotalScore(1, section, stable)]]</td>
                       <td>[[getSubtotalScore(2, section, stable)]]</td>
+                      <td>[[getSubtotalScore(3, section, stable)]]</td>
                     </tr>
                   </template>
-                </template>
-                <template is="dom-if" if="[[section.score_as_group]]">
-                  <template is="dom-repeat" items="{{section.rows}}" as="rowName">
-                    <tr>
-                      <td colspan=3>[[rowName]]</td>
-                      <td>[[getInvestigationScore(rowName)]]</td>
-                    </tr>
-                  </template>
-                  <template is="dom-if" if="[[shouldShowSubtotals()]]">
+                  <template is="dom-if" if="[[section.score_as_group]]">
+                    <template is="dom-repeat" items="{{section.rows}}" as="rowName">
+                      <tr>
+                        <td colspan=4>[[rowName]]</td>
+                        <td>[[getInvestigationScore(rowName, section.previous_investigation)]]</td>
+                      </tr>
+                    </template>
                     <tr class="subtotal-row">
-                      <td colspan=3><strong>SUBTOTAL</strong></td>
-                      <td>[[getInvestigationScoreSubtotal()]]</td>
+                      <td><strong>TOTAL</strong></td>
+                      <td colspan=3></td>
+                      <td>[[getInvestigationScoreSubtotal(section.previous_investigation)]]</td>
                     </tr>
                   </template>
                 </template>
-              </template>
-            </tbody>
-            <tfoot>
-              <tr>
-                <th><b>GRAND TOTAL</b></th>
-                <th>[[totalChromium]]</th>
-                <th>[[totalFirefox]]</th>
-                <th>[[totalSafari]]</th>
-              </tr>
-            </tfoot>
-          </table>
+              </tbody>
+            </table>
+          </div>
```

Comment:
This section changes the "SUBTOTAL" text to "TOTAL", and adds a template to display a section for previous investigation scores if any investigation scores are defined for the previous year.